Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
```

Comment:
For me `kosmtik plugins --install pluginname` worked. The only problem was with the config file.